katello_certs_tools/katello_ssl_tool.py

```diff
@@ -625,18 +662,20 @@ def genCaRpm(d, verbosity=0):
     chdir(cwd)
     _reenableRpmMacros()
 
-    out = out_stream.read(); out_stream.close()
-    err = err_stream.read(); err_stream.close()
+    out = out_stream.read()
+    out_stream.close()
+    err = err_stream.read()
+    err_stream.close()
 
     if ret or not os.path.exists("%s.noarch.rpm" % clientRpmName):
         raise GenCaCertRpmException("CA public SSL certificate RPM generation "
-                                "failed:\n%s\n%s" % (out, err))
+                                    "failed:\n%s\n%s" % (out, err))
     if verbosity > 2:
         if out:
-            print "STDOUT:", out
+            print("STDOUT:", out)
         if err:
-            print "STDERR:", err
-    os.chmod('%s.noarch.rpm' % clientRpmName, 0644)
+            print("STDERR:", err)
+    os.chmod('%s.noarch.rpm' % clientRpmName, '0644')
```

Comment:
`os.chmod` does not accept string parameters (at least not in Python 3.6 and 3.7)